testsuite/features/secondary/srv_timezone_feature

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+#  1) create a user and assing him a timezone different than the server's timezone
+#  2) test that the popups in some scheduling actions appear in users prefered timezone 
+#  3) some scheduler tests based on previous bugs are unavoidable
+
+#@sle_minion
+Feature: Correct timezone display 
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Create a new user in a timezone different than server's timezone
+    When I follow the left menu "Users > User List > Active"
+    And I follow "Create User"
+    And I enter "user1" as "login"
+    And I enter "user1" as "desiredpassword"
+    And I enter "user1" as "desiredpasswordConfirm"
+    And I select "Mr." from "prefix"
+    And I enter "Test" as "firstNames"
+    And I enter "User" as "lastName"
+    And I enter "galaxy-noise@suse.de" as "email"
+    And I select "(GMT+0800) Malaysia" from "timezone"
+    And I click on "Create Login"
+
+  Scenario: Add roles
+    When I follow the left menu "Users > User List > Active"
+    And I follow "user1"
+    When the "role_satellite_admin" checkbox should be disabled
+    And I check "role_org_admin"
+    And I check "role_system_group_admin"
+    And I check "role_channel_admin"
+    And I check "role_activation_key_admin"
+    And I check "role_config_admin"
+    And I click on "Update"
+
+  Scenario: Login as the new user
+    Given I am authorized as "user1" with password "user1"
+    Then I should see a "user1" link
+
+# bsc 1192699, a P2 bug that is now fixed
+# bsc 1195455, a P3 bug not fixed yet, 
+  Scenario: Schedule a remote script in the future and see the correct timezone as a pop up
+    Given I am on the Systems overview page of this "onalmpantis-min-sles15sp3-1.tf.local"
+    When I follow "Remote Command" in the content area
+    And I enter as remote command this script in
+      """
+      #!/bin/bash
+      ls
+      """
+    And I enter "00:00" as "date_timepicker_widget_input"
+    And I click on "Schedule"
+    Then I should see a "12:00:00 AM MYT" text
+    #WORKAROUND If the above line gets red, they probably fixed the bug, then remove AM from the text
+    
+#  Scenario: Login as the new user if the previous scenario failed
```

Comment:
Either remove or uncomment or add WORKAROUND comment explaining why this is commented out

Reply:
this scenario should be always there because if a scenario with a user and not an admin fails, then it automatically continuous as the admin. so in all my scenarios that have to do with a user, I have to make sure to have a scenario to login as the user after them, in case the previous scenario fails, and so that the rest of the scenarios can continue as user and not admin

Reply:
Therefore I will uncomment